helm-framework/helm/resource_helm_release.go

```diff
@@ -1129,7 +1129,7 @@ func chartPathOptions(d *HelmReleaseModel, meta *Meta, cpo *action.ChartPathOpti
 		chartName = u.String()
 	} else {
 		var err error
-		repositoryURL, chartName, err = resolveChartName(repository, strings.TrimSpace(chartName))
+		repositoryURL, chartName, err = buildChartNameWithRepository(repository, strings.TrimSpace(chartName))
 		if err != nil {
 			diags.AddError("Error Resolving Chart Name", fmt.Sprintf("Could not resolve chart name for repository %s and chart %s: %s", repository, chartName, err))
```

Comment:
we'll want to update the naming also in the logs since it still refers to it as `resolving chart name`